packages/ui-keyring/src/Keyring.ts

```diff
@@ -165,6 +167,12 @@ export class Keyring extends Base implements KeyringStruct {
   }
 
   private loadAddress (json: KeyringJson, key: string) {
+    const { isRecent, whenCreated = 0 } = json.meta;
+    if (isRecent && Date.now() - whenCreated > RECENT_EXPIRY) {
```

Comment:
(nitpicks) Could we just add a empty line before the if & I believe `(Date.now() - whenCreated) > RECENT_EXPIRY` reads clearer